src/controllers/still.required.controller.ts

```diff
@@ -34,13 +36,18 @@ const postRoute = async (req: Request, res: Response, next: NextFunction): Promi
 
     return renderPageWithError(res, errorText, companyName);
   } else {
-    return res.render(STILL_REQUIRED, {
-      companyName,
-      templateName: STILL_REQUIRED,
-    });
+    await addDecisionToSession(req.body.stillRequired, req.chSession);
+    const companyNumber = getPromiseToFileSessionValue(req.chSession, COMPANY_PROFILE).companyNumber;
+    const url = PROMISE_TO_FILE + "/company/" + companyNumber + APPEND_CONFIRMATION;
+    return res.redirect(url);
   }
 };
 
+const addDecisionToSession = async (decision: string, session: Session): Promise<void> => {
+  const decisionFlag: boolean = decision === "yes";
```

Comment:
I would convert decision to uppercase and check ==="YES", that way this function won't break if the template changes to Yes or YEs or yes etc..